Update books.md

pdgit25 opened the pull request:
Dear Dr. Anita 

Thank you for your reply.

Basically it's a text book which teaches QGIS use of census data to elaborate the spatial analysis of a region. ​I have created a pull request as suggested. Kindly do the needful.

Kind Regards
Pulak


On Sat, 1 Feb, 2025, 2:30 pm Anita Graser, <anita@qgis.org> wrote:

    Dear Dr. Pulak Das,

    Congratulations to your book.

    I had a look at the website and wasn't 100% sure if it is a scientific publication that uses QGIS or a textbook that teaches QGIS.

    If it is a textbook, please create a pull request for this page:

    https://github.com/qgis/QGIS-Website/blob/main/content/resources/books.md

    Regards
    Anita

nickbearman commented:
Some more comments:
I think we need to make the instructions for Ubuntu users easier, as for people new to Ubuntu / Unix, things are quite confusing to get QGIS installed. 

The "Easy Installation" section currently doesn't work. This would require QGIS to be listed in the 'Software' program in Ubuntu - does anyone have experience of doing this / is this something we can do?

The "More Detailed Installation" does currently work, and takes the user through adding universe, and adding the https://qgis.org/debian repository. I don't think we should confuse people with multiple repositories here - just say "use this one".

Common Problems is fairly self explanatory, and I will add to this in due course. 

After I have currently just left the current instructions as they are. We can tidy this up if we go ahead with the new instructions above. 

Comments, thoughts, suggestions, etc. all welcome. 

Best wishes,
Nick.

reginaldc commented:
Hi Nick,

I have read the instructions for ubuntu that you propose. These are all fine except that half of the qgis users  want to install the 2.18.21 version (as on Osgeo4w in Windows).
Could you add the procedure to do this on Ubuntu?

Thanks for this nice work,

elpaso commented:
@reginaldc just curious: can you share the source of your statistics about the 50% of users willing to install 2.18.21 version? 
Do you have a confidence interval for your figures?

reginaldc commented:
No, it's just a rough estimation based on the use of qgis in the workplace and in other municipalities I know.

rduivenvoorde commented:
@nickbearman thanks for you work, sorry for not pulling earlier.

pdgit25 committed:
Update books.md

Dear Dr. Anita 

Thank you for your reply.

Basically it's a text book which teaches QGIS use of census data to elaborate the spatial analysis of a region. ​I have created a pull request as suggested. Kindly do the needful.

Kind Regards
Pulak


On Sat, 1 Feb, 2025, 2:30 pm Anita Graser, <anita@qgis.org> wrote:

    Dear Dr. Pulak Das,

    Congratulations to your book.

    I had a look at the website and wasn't 100% sure if it is a scientific publication that uses QGIS or a textbook that teaches QGIS.

    If it is a textbook, please create a pull request for this page:

    https://github.com/qgis/QGIS-Website/blob/main/content/resources/books.md

    Regards
    Anita

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/qgis/QGIS-Website?pullRequest=542) <br/>All committers have signed the CLA.

anitagraser committed:
Update books.md

anitagraser committed:
Update books.md
